libtiledbsoma/src/soma/soma_array.cc

```diff
@@ -1470,7 +1470,7 @@ std::pair<bool, std::string> SOMAArray::_can_set_shape_helper(
                 false,
                 fmt::format(
                     "{}: array currently has no shape: please use "
-                    "tiledbsoma_upgrade_shape.",
+                    "upgrade the array.",
```

Comment:
"please use upgrade_shape" or "please upgrade the array"

Reply:
Commit # 2 https://github.com/single-cell-data/TileDB-SOMA/pull/3139/commits/3f5850bd5a4ecf2cf7385d22dad8eb40e9da94b0